navigation.yaml

```diff
@@ -149,6 +151,8 @@ managedapps:
           path: /managed/apps/kafka
         - title: Cassandra
           path: /managed/apps/cassandra
+    - title: Observability
```

Comment:
I don't think we need this here as it appears in another place and does the same thing